lib/local_authorities_importer.rb

```diff
@@ -0,0 +1,72 @@
+require 'json_downloader'
+
+class LocalAuthoritiesImporter
+  include JsonDownloader
+
+  DISTRICT = 'district'
+  COUNTY = 'county'
+  UNITARY = 'unitary'
+
+  LOCAL_AUTHORITY_MAPPING = {
+    "COI" => UNITARY,
+    "CTY" => COUNTY,
+    "DIS" => DISTRICT,
+    "LBO" => UNITARY,
+    "LGD" => UNITARY,
+    "MTD" => UNITARY,
+    "UTA" => UNITARY,
+  }
+
+  def self.import_from_mapit
+    new.authorities_from_mapit
+  end
+
+  def authorities_from_mapit
+    url = "#{MAPIT_BASE_URL}areas/#{local_authority_types}"
+    mapit_las = get_mapit_authorities(url)
```

Comment:
I don't think it is worth it there - we'll be taking that import out of Publisher in the next few months, and that's the only place where it calls MapIt directly (it goes via Imminence to get the [areas](https://github.com/alphagov/publisher/blob/30b44b7417285589484d7e0d18c76329afd2920e/app/models/area.rb) for business support schemes).